components/components/view-modes/highlighted-small/_node-highlighted-small.css

```diff
@@ -0,0 +1,70 @@
+/**
+ * @file
+ * This file is used to style the 'highlighted-small' view mode.
+ */
+
+/*
+  Highlighted Small
+
+  This presents the 'Highlighted Small' view mode.
+
+  Markup: node--highlighted-small.html.twig
+
+  Style guide: components.view_modes.highlighted_small
+*/
+
+.node--view-mode-highlighted-small {
+  border: 1px solid #fcece7;
+}
+
+.node--view-mode-highlighted-small .node__meta {
+  padding: 1.5rem;
+}
+
+.node--view-mode-highlighted-small .node__title {
+  margin-bottom: 1rem;
+}
+
+.node--view-mode-highlighted-small .field--name-title {
+  font-family: 'Scope One';
+  font-size: 1.424rem;
+}
+
+.node--view-mode-highlighted-small .node__title a {
+  color: #000;
+  text-decoration: none;
+}
+
+.node--view-mode-highlighted-small .node__title a:focus,
+.node--view-mode-highlighted-small .node__title a:hover, {
```

Comment:
It simply underlined on focus/hover. This is the desired behavior? Eg the more link changes its color but not underlined.

Reply:
Oh this is defined some lines below (l48, l49). So it is by design?

Reply:
We have no design for hover state of the title, so I put in an underline (for accessibility reasons, colour change is not enough).

Reply:
Next version of the styleguide pdf resolves this. I've added some screenshots of the relevant parts in Slack

Reply:
I am leaving the underline as is for now. We can't change the color of the underline with CSS (we can with CSS 3, but it's not supported by the browser range we need to support).

We could use a bottom border, but that has issues then with line-height, leaving it like this:
![screen shot 2017-10-14 at 13 36 54](https://user-images.githubusercontent.com/2183332/31575715-cb19e73c-b0e4-11e7-9d3e-54f17a2fcb13.png)

Reply:
I'm going to bet that every idea we come up with as a new approach will rely on css3 support! Let's discuss in the channel as this has a knock-on I'm guessing